product_pricelist_simulation/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Product Pricelist Simulation",
+    "summary": "Simulate the product price for all pricelists",
+    "version": "13.0.1.0.0",
+    "category": "Product",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "GRAP, Tecnativa, Odoo Community Association (OCA)",
```

Comment:
also the maintainer key has been lost.

https://github.com/OCA/margin-analysis/blob/12.0/product_pricelist_margin/__manifest__.py#L10